.ado/jobs/e2e-test.yml

```diff
@@ -62,34 +62,12 @@ jobs:
           script: yarn windows --no-launch --no-packager --arch $(BuildPlatform) --release --logging --buildLogDirectory $(BuildLogDirectory) --msbuildprops BaseIntDir=$(BaseIntDir)
           workingDirectory: packages/E2ETest
 
-      - task: CopyFiles@2
-        displayName: Copy ReactUWPTestApp artifacts
-        inputs:
-          sourceFolder: $(Build.SourcesDirectory)/packages/E2ETest/windows/ReactUWPTestApp
-          targetFolder: $(Build.StagingDirectory)/ReactUWPTestApp
-          contents: AppPackages\**
-        condition: succeededOrFailed()
-
-      - task: PublishPipelineArtifact@1
-        displayName: "Publish Artifact:ReactUWPTestApp"
-        inputs:
-          artifactName: ReactUWPTestApp-$(System.JobAttempt)
-          targetPath: $(Build.StagingDirectory)/ReactUWPTestApp
-        condition: succeededOrFailed()
-
       - task: CmdLine@2
         displayName: run e2etest
         inputs:
           script: yarn run e2etest
           workingDirectory: packages/E2ETest
 
-      - task: CopyFiles@2
-        displayName: Copy test report
-        inputs:
-          sourceFolder: $(Build.SourcesDirectory)\packages\E2ETest\reports
-          targetFolder: $(Build.StagingDirectory)/ReactUWPTestAppTreeDump/reports
-        condition: succeededOrFailed()
-
```

Comment:
Jest does support reporters, but we weren't really using the reports for much of anything, so I haven't hooked them up yet. Can do so if we have a need, but otherwise seems like a nice opportunity to reduce complexity.

Reply:
At some point in time it would be nice ot hook them up as well as coverage reports so that we can report them to azure devops. and maybe at som epoint start a policy that new code must be covered.

Reply:
+1 I want to add coverage reporters to shared jest config for CLI apps at least. Probably less useful for E2ETest because we're testing something out of process but we could maybe do something for reporters without coverage.